### PR TITLE
Changes for deploying the Application

### PR DESCRIPTION
**Pull request by saruniitr**
## Change description

This PR adds changes to deploy the application to Paas Environment. In this case initially dev environment is chosen.

The app is currently deployed to dev environment using this changes and is available at,
https://enquiry-mgmt-dev.london.cloudapps.digital/

### CODE REVIEW

File: app/settings/local.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': env('POSTGRES_DB_NAME', default='postgres'),
+        'USER': env('POSTGRES_DB_USER', default='postgres'),
+        'PASSWORD': env('POSTGRES_DB_PASSWORD'),
+        'HOST': env('POSTGRES_DB_HOST'),
+        'PORT': env.int('POSTGRES_DB_PORT', default=5432),
+    }
+}
```
Comment (reupen) [inline app/settings/local.py]: Could local use `DATABASE_URL` as well?

File: manifest.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+applications:
+-
+  buildpack:
```
Comment (reupen) [inline manifest.yml]: Is the newline after `-` needed?
```suggestion
- buildpack:
```

File: web.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+# Start-up script for the web process, primarily for GOV.UK PaaS (see the Procfile)
+
+set  -xe
+
+# Compile css
+npm rebuild node-sass && npm install --only=dev && npm run sass
+
+./manage.py migrate --noinput
+./manage.py collectstatic --noinput
```
Comment (reupen) [inline web.sh]: Normally the buildpack runs `collectstatic` (unless `DISABLE_COLLECTSTATIC` is set) – was that not working?

File: app/settings/local.py (hunk 1 above)
Comment (saruniitr) [reply app/settings/local.py]: yes but I still need to provide host, user etc so thought its easier to keep it like this.

File: manifest.yml (hunk 2 above)
Comment (saruniitr) [reply manifest.yml]: not really, initially some syntax errors were causing weird issues so just followed as in example. Will remove it if not required.

File: web.sh (hunk 3 above)
Comment (saruniitr) [reply web.sh]: It is really not necessary as the `gulp` script is placing the files in correct directory, just added this for completeness as it shouldn't have any effect, will remove it.

File: manifest.yml (hunk 2 above)
Comment (saruniitr) [reply manifest.yml]: fixed

File: app/settings/dev.py [hunk 4]
```diff
@@ -0,0 +1 @@
+from app.settings.common import *
```
Comment (reupen) [inline app/settings/dev.py]: Is this settings module really needed?

File: app/settings/local.py [hunk 5]
```diff
@@ -0,0 +1 @@
+from app.settings.common import *
```
Comment (reupen) [inline app/settings/local.py]: Is this settings module really needed?

File: app/settings/dev.py (hunk 4 above)
Comment (saruniitr) [reply app/settings/dev.py]: well it is not required at the moment but left it for future requirement.
I guess I can add it back when required, will remove it now.

File: app/settings/local.py (hunk 5 above)
Comment (saruniitr) [reply app/settings/local.py]: same as above

**Comment by reupen** [comment]
> Could you clarify what you mean tidy up commits?

Usually we squash unnecessary commits before merging. In this case, there are 31 commits, but it could probably be somewhere between 1 and 3 commits. So, I mean, once you have a second approval, it would be good to rebase and squash some of the commits.

**Comment by saruniitr** [comment]
IMHO I think it is better to leave as is, if we squash we lose history. The large number of commits in branch is main because we can only test the changes after deployment. If squashing is preferred I can merge using "Squash and merge" option.

**Comment by saruniitr** [comment]
@alixedi I have addressed all comments, waiting for the second approval to merge.

**Comment by saruniitr** [comment]
Thanks for your reviews, received two approvals, merging it.

**Pull request merged**